KBH/2Day/가운데 글자 가져오기.js

```diff
@@ -0,0 +1,7 @@
+function solution(s) {
+    let answer = '';
+    const slice = s.length / 2;
+    //2로 나누어서 짝수, 홀수 구분하기위한 상수 선언
+    answer = (s.length % 2) === 0 ? s.substr(slice-1,2) : s.substr(slice ,1);
+    return answer;
+}
```

Comment:
```suggestion
function solution(s) {
    const middleIdx = s.length / 2;
    let answer = '';
    
    //2로 나누어서 짝수, 홀수 구분하기위한 상수 선언
    answer = (s.length % 2) === 0 ? s.substr(slice-1, 2) : s.substr(slice, 1);
    return answer;
}
```

@KimBYH 
- 기본적으로 상수를 의미하는 const가 let보다 상단에 올라와야 됩니다 :)
- slice라는 네이밍이 명확하지 않은 것 같아요. s문자열의 중앙 인덱스를 의미하는 middleIndex와 같은 명확한 네이밍 사용이 어떨까요?

Reply:
@ssi02014 
1. 상수를 선언할 때 let보다 상단에 올라가서 선언할 수 있도록 하겠습니다.
2. middleIndex의 네이밍이 다른 사람들이 보기에 잘 이해할 수 있는 네이밍인것 같습니다.